docs/installation/intrusion_detection_system_integration.asciidoc

```diff
@@ -212,6 +212,105 @@ Configuration of a new security event can use the following trigger types:
   Type: suricata_event
   Trigger ID: The rule class of the triggered IDS alert
 
+=== Security Onion 2.3.10
+
+Security Onion (SO) software has a new version, 2.3. You can review the documentation at: https://docs.securityonion.net/en/2.3/
```

Comment:
I don't think we should say something like that - in a sense it doesn't bring any value.

Just say: "This documentation is based on Security Onion v2.3.  You can review its documentation at: https://docs.securityonion.net/en/2.3"